handlers/jiochat/handler_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	"log/slog"
```

Comment:
no space needed between this and other deps - standard go formatting is only a space between std lib deps and other deps